metaseq/trainer.py

```diff
@@ -1002,7 +1002,7 @@ def clip_grad_norm(
             clip_norm,
             clip_norm_type,
             aggregate_norm_fn=None,
-            skip_gradient_update_on_clip_norm=skip_gradient_update_on_clip_norm,
+            # skip_gradient_update_on_clip_norm=skip_gradient_update_on_clip_norm,
```

Comment:
Nit: probably don't want to hardcode this commented out flag.

Reply:
i'll remove it entirely as this is an unknown argument